### PR TITLE
fix: file_format validate() for field_optionally_enclosed_by

### PR DESCRIPTION
**Pull request by raulbonet**
ref #2385 

As pointed out by https://github.com/alexmaras , the `validate()` function does not properly validate the use-case where field_optionally_enclosed_by is None.

### CODE REVIEW

**Comment by raulbonet** [comment]
@sfc-gh-asawicki 
I wonder if the `validation()` function should go as far as validating potential values that Snowflake will already validate when the query is executed. Not sure if it would be best just to remove all together this part of the validate. If tomorrow Snowflake adds other accepted values to this list, it would manually need to be modified.

File: pkg/sdk/file_format.go [hunk 1]
```diff
@@ -572,7 +572,7 @@ func (opts *FileFormatTypeOptions) validate() error {
 		return fmt.Errorf("IgnoreUTF8Errors and ReplaceInvalidCharacters cannot be set simultaneously")
 	}
 
-	validEnclosedBy := []string{"null", "'", `"`}
+	validEnclosedBy := []string{"NONE", "'", `"`}
```
Comment (sfc-gh-asawicki) [inline pkg/sdk/file_format.go:575]: It would be nice to see this value used in one of the SDK integration tests (`file_format_integration_test.go`).

**Comment by sfc-gh-asawicki** [comment]
> @sfc-gh-asawicki I wonder if the `validation()` function should go as far as validating potential values that Snowflake will already validate when the query is executed. Not sure if it would be best just to remove all together this part of the validate. If tomorrow Snowflake adds other accepted values to this list, it would manually need to be modified.

We currently have this discussion when approaching resources redesign. Our SDK-level validations or sometimes to strict and it opens us to the problem you mentioned. We will probably soften the validations on the general level but there is no decision yet.

**Comment by sfc-gh-asawicki** [comment]
Thanks for the PR @raulbonet. I left one comment.

File: pkg/sdk/file_format.go (hunk 1 above)
Comment (raulbonet) [reply pkg/sdk/file_format.go:575]: added integration test

**Review by sfc-gh-asawicki — APPROVED** [review]

**Comment by sfc-gh-asawicki** [comment]
Thanks for the contribution @raulbonet! 🎉

**Pull request merged**